ibis/backends/dask/tests/test_core.py

```diff
@@ -177,3 +174,9 @@ def test_scope_look_up():
     scope = scope.merge_scope(Scope({one_day: 1}, None))
     assert scope.get_value(one_hour) is None
     assert scope.get_value(one_day) is not None
+
+
+def test_new_dispatcher():
+    types = (ops.TableColumn, dd.DataFrame)
+    assert execute_node.dispatch(*types) is not None
+    assert pandas_execute_node.dispatch(*types) is None
```

Comment:
Should this be `dask_execute_node.dispatch(*type) is None`?

Reply:
Here, we're using a dask DataFrame, so execute_node (the dask dispatcher) should have the function, where the pandas one doesn't.

Reply:
Oh nvm. I misread "dd" to be "pd". They are so close XD